Added a security policy

Koen1999 opened the pull request:
Please note that this is a draft. We can use the PR to discuss and improve before accepting this policy.

Koen1999 commented on SECURITY.md:
```diff
@@ -0,0 +1,35 @@
+
+## Reporting a Vulnerability
+
+If you would like to report a vulnerability, mention this to [web@gewis.nl](mailto:web@gewis.nl).
```
The main reason I didn't include a PGP key yet is because I think it is difficult to share a PGP key amongst several members while we are constantly getting new members and having others depart. I think once we are contacted, we can share a temporary PGP key if necessary.

kokx replied:
Well, my suggestion includes having a smaller list of people dealing with security issues anyway. Its not needed to share issues with the entire web, and actually its better to have this be a small and more stable group.

Koen1999 commented on SECURITY.md:
```diff
@@ -0,0 +1,35 @@
+# Security Policy
+
```
Good one

kokx commented on SECURITY.md:
```diff
@@ -0,0 +1,35 @@
+## Supported Versions
+
+Only vulnerabilities in the most recently released version and in the version currently on the master branch are considered to be significant.
+If you discover a vulnerability in an earlier version that may still be present in the most recently released version, you can still report it.
```
No, but we can write that the version deployed on gewis.nl is covered.

kokx commented on SECURITY.md:
Either development version, or deployment of the website

kokx commented on SECURITY.md:
I would make this something like "Testing on the website is however not allowed."

tomudding reviewed (commented):
I agree with what Pieter has suggested, setting up a separate group (possibly also with someone from the CBC) with sole access to a custom security mailbox and PGP key.

While `sexurity.txt` is not yet a standard, all recent drafts require the document to be [located in `/.well-known/`](https://datatracker.ietf.org/doc/html/draft-foudil-securitytxt-12#section-4). This is currently not the case, I would therefore also recommend serving the document directly with NGINX (by adding a new `location` directive).

Furthermore, the [field `Expires`](https://datatracker.ietf.org/doc/html/draft-foudil-securitytxt-12#section-3.5.5) is missing, which is a required field.

Koen1999 commented:
> 
> 
> I agree with what Pieter has suggested, setting up a separate group (possibly also with someone from the CBC) with sole access to a custom security mailbox and PGP key.
> 
> While `sexurity.txt` is not yet a standard, all recent drafts require the document to be [located in `/.well-known/`](https://datatracker.ietf.org/doc/html/draft-foudil-securitytxt-12#section-4). This is currently not the case, I would therefore also recommend serving the document directly with NGINX (by adding a new `location` directive).
> 
> Furthermore, the [field `Expires`](https://datatracker.ietf.org/doc/html/draft-foudil-securitytxt-12#section-3.5.5) is missing, which is a required field.

I added an expiration date not too far in the future and moved the file. It is already served by nginx directly.

Koen1999 commented:
While I think it's a good idea to have a small and stable group, I foresee difficulties with this given we are all volunteers basically.

tomudding reviewed (approved):
Requested changes have been made, so everything looks good. With regard to the security group, we have discussed this with Rink and he will (try to) arrange this with the CBC.
